LGTM! Just seeing in the tests
```
00:21:52 poseidon int  :	15335/96500 (mem: 74.61 MiB)
00:21:52 overflow      :	11/11 (mem: 74.61 MiB)
00:21:52  (mem: 74.61 MiB)
00:21:52 Total structured size: 260012 (mem: 74.61 MiB)
00:21:53 Contract postprocessing complete!
00:21:57 file:///usr/src/yarn-project/native/dest/native_module.js:16
00:21:57         throw new Error(`Failed to load native module for ${variant} from ${modulePath}. ` + `Supported: amd64-linux, arm64-linux, amd64-macos, arm64-macos. ` + `Error: ${error}`);
00:21:57               ^
00:21:57 
00:21:57 Error: Failed to load native module for amd64-linux from /usr/src/yarn-project/native/build/amd64-linux/nodejs_module.node. Supported: amd64-linux, arm64-linux, amd64-macos, arm64-macos. Error: Error: Cannot find module '/usr/src/yarn-project/native/build/amd64-linux/nodejs_module.node'
00:21:57 Require stack:
00:21:57 - /usr/src/yarn-project/native/dest/native_module.js
00:21:57     at loadNativeModule (file:///usr/src/yarn-project/native/dest/native_module.js:16:15)
00:21:57     at file:///usr/src/yarn-project/native/dest/native_module.js:19:22
00:21:57     at ModuleJob.run (node:internal/modules/esm/module_job:345:25)
00:21:57     at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:665:26)
00:21:57     at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:117:5)
00:21:57 
00:21:57 Node.js v22.20.0
00:21:57 Dumping docker logs...
```

--- a/release-image/Dockerfile.dockerignore
+++ b/release-image/Dockerfile.dockerignore
@@ -23,7 +23,7 @@
 !/yarn-project/protocol-contracts/artifacts/
 !/yarn-project/noir-contracts.js/artifacts/
 !/yarn-project/simulator/artifacts/
-!/yarn-project/native/build/nodejs_module.node
+!/yarn-project/native/build/
 
 # Needed to generate list of env vars to know what to be able to inject into the container.
 !/yarn-project/foundation/src/config/env_var.ts

